### ISSUE TITLE
Documentation improvement

### ISSUE THREAD

**Issue by gjain-7** [hyperopt/hyperopt#889]
In the **Getting Started** section of the [docs](http://hyperopt.github.io/hyperopt) the snippet 
```python
# define an objective function
def objective(args):
    case, val = args
    if case == 'case 1':
        return val
    else:
        return val ** 2

# define a search space
from hyperopt import hp
space = hp.choice('a',
    [
        ('case 1', 1 + hp.lognormal('c1', 0, 1)),
        ('case 2', hp.uniform('c2', -10, 10))
    ])

# minimize the objective over the space
from hyperopt import fmin, tpe
best = fmin(objective, space, algo=tpe.suggest, max_evals=100)

print best
# -> {'a': 1, 'c2': 0.01420615366247227}
print hyperopt.space_eval(space, best)
# -> ('case 2', 0.01420615366247227}
```

throws the error `NameError: name 'hyperopt' is not defined` at the second last line.

A quick import may be useful
```python 
import hyperopt
```

although not a big issue, but definitely an improvement in the docs.

**Comment by maxpumperla** [comment]
@gjain-7 this is generated directly from the main readme. want to open a quick PR?

**Comment by gjain-7** [comment]
Sure. Will do it in few hours.
btw, Is there anything else to add/improve in this or any related doc ?

**Comment by gjain-7** [comment]
Hey @maxpumperla just noticed that this was already fixed in c903de9. https://github.com/hyperopt/hyperopt/blob/c903de979e50f844130dcc08107038f95e87a061/README.md?plain=1#L43

Looks like the latest [gh page deployment](https://github.com/hyperopt/hyperopt/deployments/activity_log?environment=github-pages) is stuck to https://github.com/hyperopt/hyperopt/commit/35c5c14c686fe4235cb5dfe057a24a50b5a3055a.

**Comment by gjain-7** [comment]
Hey @maxpumperla, any update on this ?
